app/app.go

```diff
@@ -91,12 +91,12 @@ import (
 	appparams "github.com/umee-network/umee/app/params"
 	uibctransfer "github.com/umee-network/umee/x/ibctransfer"
 	uibctransferkeeper "github.com/umee-network/umee/x/ibctransfer/keeper"
-	"github.com/umee-network/umee/x/leverage"
-	leveragekeeper "github.com/umee-network/umee/x/leverage/keeper"
 	leveragetypes "github.com/umee-network/umee/x/leverage/types"
 	"github.com/umee-network/umee/x/peggy"
 	peggykeeper "github.com/umee-network/umee/x/peggy/keeper"
 	peggytypes "github.com/umee-network/umee/x/peggy/types"
+	// "github.com/umee-network/umee/x/leverage"
```

Comment:
We want to disable/not include any incomplete module in the app as we'll be getting ready for an official mainnet release soon.

Reply:
I've added a `beta` package that we can use for integration and client tests in the meantime